service/src/main/java/bio/terra/externalcreds/services/PassportService.java

```diff
@@ -37,6 +41,8 @@ public class PassportService {
   private final JwtUtils jwtUtils;
   private final Collection<VisaComparator> visaComparators;
 
+  private static final long VISA_VALIDITY_TIME = Duration.of(1, ChronoUnit.HOURS).toMillis();
```

Comment:
It might be easier to maintain if this is just a `Duration`

Reply:
Then we'd be calling `toMillis()` every time we need to validate an external passport.

Reply:
Yeah, I just feel like it's more obvious what it is if it's a duration.  At the very least, it should probably be named `VISA_VALIDITY_TIME_MILLIS` (also, I don't think the `toMillis()` call is expensive but your call

Reply:
I sidestepped the issue by actually utilizing the `Duration` library to do the comparison for me!